### PR TITLE
perf: adjust scaffold folder structure

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: packages/fx-core/.vscode/launch.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
         "ts-node/register",
         "--no-timeouts",
         "--colors",
-        "${workspaceFolder}/tests/**/*.test.ts"
+        "${workspaceFolder}/tests/plugins/resource/bot/unit/generateArmTemplates.test.ts"
```
Comment (dilin-MS) [inline packages/fx-core/.vscode/launch.json:14]: What's this for?

Comment (xzf0587) [inline packages/fx-core/.vscode/launch.json:14]: update the code to mocha all

File: packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep [hunk 2]
```diff
@@ -26,6 +25,6 @@ resource botWebAppSettings 'Microsoft.Web/sites/config@2021-01-15' = {
     M365_CLIENT_SECRET: m365ClientSecret
     M365_TENANT_ID: m365TenantId
     M365_APPLICATION_ID_URI: m365ApplicationIdUri
-    IDENTITY_ID: provisionOutputs.identityOutput.value.resourceId
+    IDENTITY_ID:
```
Comment (blackchoey) [inline packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep:28]: the value is empty here

File: packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep [hunk 3]
```diff
@@ -7,7 +7,6 @@ param provisionOutputs object
 param currentAppSettings object
 
 var botWebAppName = split(provisionOutputs.botOutput.value.webAppResourceId, '/')[8]
-
 var m365ClientId = provisionParameters['m365ClientId']
```
Comment (xzf0587) [inline packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep:10]: this file name contains "Result", not "result.
Update it to keep consistant

Comment (wenytang-ms) [reply packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep]: updated.

File: packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep (hunk 2 above)
Comment (wenytang-ms) [reply packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep]: updated.

File: packages/fx-core/.vscode/launch.json (hunk 1 above)
Comment (wenytang-ms) [reply packages/fx-core/.vscode/launch.json]: reverted

Comment (wenytang-ms) [reply packages/fx-core/.vscode/launch.json]: reverted

File: packages/fx-core/src/plugins/resource/identity/constants.ts [hunk 4]
```diff
@@ -46,7 +46,7 @@ export class IdentityBicep {
   static readonly identityClientId: string =
     "provisionOutputs.identityOutput.value.identityClientId";
   static readonly identityResourceId: string =
-    "userAssignedIdentityProvision.outputs.identityResourceId";
```
Comment (wenytang-ms) [inline packages/fx-core/src/plugins/resource/identity/constants.ts:49]: @xzf0587  this constants value needs to be updated, right ?

Comment (xzf0587) [reply packages/fx-core/src/plugins/resource/identity/constants.ts:49]: yes. please update it.

File: packages/fx-core/tests/plugins/solution/solution.scaffold.test.ts [hunk 5]
```diff
@@ -234,11 +234,8 @@ describe("Solution scaffold() reading valid manifest file", () => {
         path.join(testFolder, "./templates/azure/provision", "simpleAuthProvision.bicep")
       )
```
Comment (xzf0587) [inline packages/fx-core/tests/plugins/solution/solution.scaffold.test.ts:235]: update frontendHosting and simpleAuth file name as identity

Comment (wenytang-ms) [reply packages/fx-core/tests/plugins/solution/solution.scaffold.test.ts:235]: later do it

**Pull request merged**